ta/trusted_key/Android.mk

```diff
@@ -0,0 +1,3 @@
+LOCAL_PATH := $(call my-dir)
```

Comment:
`Android.mk` and `Makefile` are not used, I suggest to remove them (same for other in-tree early TAs but out of scope for this PR).